frontend/libs/ui/src/components/Entity/EntityItemIndexPage.tsx

```diff
@@ -35,27 +37,37 @@ export const EntityItemIndexPage = ({
   const [error, setError] = useState('');
   const dispatch = useDispatch();
   const Viewer = entityViewer;
-
+  
   useMemo(() => {
     if (!entityState.isFetched && !entityState.isFetching) {
       dispatch(entityStateProvider.loadState());
     } else {
-      if (!item && entityState.response) {
-        const list = [] as TenantedEntity[];
-        if (_.isArray(entityState.response)) {
-          list.push(...entityState.response);
-        } else {
-          list.push(...entityState.response.content);
-        }
-        const found = list.filter((k) => k.itemId === itemId);
-        if (found.length > 0) {
-          setItem(found[0]);
-        } else {
-          setError('Item not found');
-        }
-      } else if (entityState.error) {
-        setError(entityState.error);
+      if (!item){
+        entityService.getById(itemId)
+        .then((response:any) => {
+          setItem(response.data);
+          setError('');
+        })
+        .catch((err:any) => {
+          setError('Item not found..');
+        });
       }
+      // if (!item && entityState.response) {
+      //   const list = [] as TenantedEntity[];
+      //   if (_.isArray(entityState.response)) {
+      //     list.push(...entityState.response);
+      //   } else {
+      //     list.push(...entityState.response.content);
+      //   }
+      //   const found = list.filter((k) => k.itemId === itemId);
+      //   if (found.length > 0) {
+      //     setItem(found[0]);
+      //   } else {
+      //     setError('Item not found');
+      //   }
+      // } else if (entityState.error) {
+      //   setError(entityState.error);
+      // }
```

Comment:
Let's just delete this commented code.  If we need to access it again, we can view it in the git history